src/parser.ts

```diff
@@ -662,6 +672,165 @@ const TRAVERSE_BUILDER: MarkVisitor<(rhs: TraversalResult | null) => TraversalRe
   },
 }
 
+const SELECTOR_BUILDER: MarkVisitor<null> = {
+  square_bracket(p) {
+    p.process(TRAVERSE_BUILDER)
+    return null
+  },
+
+  slice(p) {
+    p.process(TRAVERSE_BUILDER)
+    return null
+  },
```

Comment:
These two are also never called at the moment. Same reason as with `attr_access`.

Reply:
I had these in to catch an error with traversals, but the problem is resolved when I process traversals until the `traversal_end` mark.